### DISCUSSION TITLE
Arabic Translation

### DISCUSSION THREAD

**Discussion by Abdelaziz18003** [vuejs-translations/guidelines#60] [General]
أهلا بالجميع،
لاحظت وجود مناقشة حول اللغة العربية هنا من قبل، لكن نشاطها يبدو قد توقف ورابط غرفة المحادثة لم يعد يعمل، لهذا أنشأت هذه المناقشة الجديدة للتعاون على ترجمة الموقع الرسمي ل Vue.js الذي بدأت العمل عليه قبل فترة وجيزة، والذي أعلم يقينا أني لن أستطيع إكماله بمفردي، فالترجمة صعبة والموقع ضخم والعمل يحتاج إلى تظافر الجهود.

إليكم روابط المستودع والموقع وغرفة الدردشة التي نسعد بانضمامكم إليها، وسوف نكون متواجدين فيها لتقديم المساعدة الضرورية

**المستودع:** https://github.com/Abdelaziz18003/vuejs-docs-ar
**غرفة المحادثة:** https://discord.gg/xGbJdEz6JW (إذا واجهتكم مشكلة في الدخول إلى الغرفة فاتركوا تعليقا في الأسفل)
**الموقع العربي:** https://vuejs-docs-ar.netlify.app

فلتتكاتف جهودنا ولنمضي قدما في هذه المهمة الجليلة 💪

**Comment by itsmolaboune** [comment]
شكرا أخي!
فقط غرفة المحادثة لاتعمل

**Reply by boussadjra** [reply]
https://discord.gg/fhEj5nBRaR  جرب هذا الرابط

**Comment by faresdjefaflia** [comment]
> أهلا بالجميع، لاحظت وجود مناقشة حول اللغة العربية هنا من قبل، لكن نشاطها يبدو قد توقف ورابط غرفة المحادثة لم يعد يعمل، لهذا أنشأت هذه المناقشة الجديدة للتعاون على ترجمة الموقع الرسمي ل Vue.js الذي بدأت العمل عليه قبل فترة وجيزة، والذي أعلم يقينا أني لن أستطيع إكماله بمفردي، فالترجمة صعبة والموقع ضخم والعمل يحتاج إلى تظافر الجهود.
> 
> إليكم روابط المستودع والموقع وغرفة الدردشة التي نسعد بانضمامكم إليها، وسوف نكون متواجدين فيها لتقديم المساعدة الضرورية
> 
> **المستودع:** https://github.com/Abdelaziz18003/vuejs-docs-ar **غرفة المحادثة:** https://discord.gg/tBYASVZk (إذا واجهتكم مشكلة في الدخول إلى الغرفة فاتركوا تعليقا في الأسفل) **الموقع العربي:** https://vuejs-docs-ar.netlify.app
> 
> فلتتكاتف جهودنا ولنمضي قدما في هذه المهمة الجليلة muscle

رابط الدعوة لا يعمل

**Comment by boussadjra** [comment]
@frs99 الروابط السابقة كانت لها مدة صلاحية، لقد انشأت رابط دعوة جديد صالح للأبد 
https://discord.gg/fhEj5nBRaR

**Reply by faresdjefaflia** [reply]
شكرا لك

**Reply by boussadjra** [reply]
العفو

**Comment by moHaHa** [comment]
😁✌مرحبا 
عندي سؤال

**Reply by Abdelaziz18003** [reply]
مرحبا بك، تفضل 🙂

**Reply by moHaHa** [reply]
شو عملتو مشان موضوع ال rtl ؟
وثيقة الـVue مبنية على vitepress 
لما دخلت اشوف التعريب لقيت في اجزاء متل الـ  layout ما في عليها تغيير باتجاه اتجاه اللغة

**Reply by boussadjra** [reply]
@moHaHa حاليا قمنا بتغيير الاتجاه فقط داخل المحتوى و الشريط الجانبي من خلال تنسيق CSS لكن هناك ميزة تجريبية في vitepress لدعم الـRTL، شخصيا لا أعتبر أنها أولوية، هدفنا التركيز على ترجمة نسخة أولية قابلة للنقاش والتعديل. 
نتشرف بمساهمتك.

**Reply by moHaHa** [reply]
كيف غيرتم الاتجاه داخل المحتوى انا دخلت vitepress docs 
كل يلي لقيته هو هاد الكلام 
![image](https://user-images.githubusercontent.com/72744132/236321966-6bbe3a0f-dbbd-4b66-b9f6-e9d85d59ec09.png)

يعني الاتجاه rtl مانو مدعوم بشكل رسمي لهيك انا بسأل كيف عدلتو المحتوى ..   بناء على الكلام في vitepress ، ام عملتو حل من عندكم  
...و ان كان غير كامل فـ لا يشمل بعض الإجزاء زي الـ layout

**Reply by boussadjra** [reply]
يمكنك الدخول على المستودع و رؤية التعديلات التي قمنا بها من أجل تغيير الاتجاه

**Reply by moHaHa** [reply]
>  بسأل كيف عدلتو المحتوى .. بناء على الكلام في 
>vitepress ،
 > ام عملتو حل من عندكم ?

يعني انتو عملتو تعديل يناسب الشي 
مش معتمدين كلام شي من فيتبريس 
صح ؟

**Reply by moHaHa** [reply]
تمام لقيت ملفات overrides 
بس حبيت اتأكد قبل ما اكتب هي في  
vuejs/vitepress discussions 


[vuejs/vitepress/discussions : How does vitepress support RTL language direction ?](https://github.com/vuejs/vitepress/discussions/2336 )

**Comment by khaledOghli** [comment]
مشكور اخي الكريم يوجد العديد من الاخطاء ولكن اهمها لا تقم بترجمة المصطلحات 
ك composition api  وال option api وغيرها اتركها كما هي .حتى بجميع اللغات المدعومة تبقى كما هي .

**Reply by Abdelaziz18003** [reply]
العفو، لا يوجد هناك اتفاق على ذلك
مثلا هذه العبارة "TypeScript with Composition API " تترجم إلى:

الترجمة الأكرانية: TypeScript з Композиційним API
الترجمة البرتغالية: TypeScript com a API de Composição

**Comment by boussadjra** [comment]
> مشكور اخي الكريم يوجد العديد من الاخطاء ولكن اهمها لا تقم بترجمة المصطلحات ك composition api وال option api وغيرها اتركها كما هي .حتى بجميع اللغات المدعومة تبقى كما هي .
العفو وشكرا على اطلاعك، نعم يوجد الكثير من الأخطاء، لكن بالنسبة للمصطلحات فمن الأفضل ترجمتها إلى العربية حتى تكون مرجعا، لأن تكرارها باللفظ الإنجليزي يفقد النص العربي جماليته. قد تبدو الترجمة غريبة نوعا ما في الأول لكن مع الوقت تصير مألوفة. ولابد لنا من الاستفادة من سعة اللغة العربية بدل التضييق على أنفسنا.
على كل حال الترجمة الحالية للتوثيق تحتاج الكثير من المراجعة حتى ترقى إلى المستوى.

**Reply by khaledOghli** [reply]
لكن الا تعتقد انه من الافضل عدم تضييع الشخص الذي يقرأ ويتعلم بمصطلحات لن تمر معه، في النهاية لن يتم سؤاله في المقابلات العمل او العمل عن المصطلح باللغة العربية .
سيسمع المصطلح الاصلي ويجب ان يكون مألوف له.

**Reply by Abdelaziz18003** [reply]
الإجابة على سؤالك تختلف بحسب السياق الذي نتحدث فيه وبحسب الهدف الذي ننشده من خلال الترجمة.

إذا كان الهدف من ترجمة التوثيقات هو هدف قصير المدي نسعى من خلاله لتسهيل وصول المبرمجين للأدوات البرمجية، فاعتراضك هنا يمكن أن يكون له وجه ويمكن تفهمه، لأن ترجمة المصطلحات سيكون عبئا إضافيا على القارئ وسيكون الأسهل له ولنا كمترجمين أن نبقي المصطلحات على حالها. لكن هكذا قد يتعترض آخر ويقول ما الفائدة من الترجمة إطلاقا؟ المبرمج أغلب عمله سيكون باللغة الإنجليزية وسيبحث في الأنترنت عن حلول مشاكله البرمجية باللغة الإنجليزية فلماذا يقرأ التوثيق بالعربية مادام مضطرا لتعلم الإنجليزية والتعامل بها؟!

أما إذا كان الهدف استراتيجيا بعيد المدى مثل الإرتقاء باللغة العربية وجعلها لغة علم وتقنية، فترجمة المصطلحات لا مناص منها، رغم العبئ الثقيل الذي يلقى على كاهل المترجم الذي يتمثل في البحث عن المصطلحات الجديدة بالغوص في المعاجم العربية وتخير الأنسب منها للسياق، النتيجة ستكون توثيقا عربيا بلغة سليمة اللهم إلا المصطلحات الجديدة كليا فيمكن الإشارة إلى اللفظ الإنجليزي المقابل لها لتسهيل فهم المعنى المقصود، بعد انتشار الترجمة العربية ستصير الترجمات مفهومة ولا تحتاج لشرحها، مثل السيارة والطائرة والبرمجة والحاسوب وغيرها من المصطلحات التي تُرجمت في السابق ومع اعتياد الناس عليها صارت ملتصقة بمدلولها لا تفارقه ولا تحتاج لشرح ولا لاستحضار اللفظ الانجليزي الأصلي الذي ترجمت منه. هذا ما نصبوا إليه فلهذا نترجم المصطلحات

**Comment by Abdelaziz18003** [comment]
Hi @Jinjiang,
This is the Arabic translation original discussion. We've translated all the guide and a good portion of the API pages. We think that this should be more than 50% of the content which makes this translation eligible to be added to `vuejs-translations` organization as mentioned in the Vue Translation Guidelines. Please let me know how we can proceed.
Regards.

**Reply by Jinjiang** [reply]
@Abdelaziz18003 great job! I have sent to you an invitation to proceed. Thanks.

Please remember to detach your fork before the transfer.

**Reply by Abdelaziz18003** [reply]
@Jinjiang thanks for the invitation. I have detached the repository and transfered it to this organization under the name of `docs-ar`. Here is the new repo https://github.com/vuejs-translations/docs-ar

**Reply by Abdelaziz18003** [reply]
@Jinjiang when reading the guidelines, I had the impression that getting the translation specific subdomain (ex: https://ar.vuejs.org in our case) will be the next step after the repo transfer. I am not sure if I got it right and this step will be done in the upcoming days by one of the maintainers or the translation needs to be completed first before claiming that subdomain?

**Reply by Jinjiang** [reply]
@Abdelaziz18003 you are right. The subdomain is under preparation. We will let you know once it's done. You can feel free to proceed the translation process. Thanks again for your great contribution!

**Reply by Abdelaziz18003** [reply]
@Jinjiang thanks, I appreciate your response

**Reply by Jinjiang** [reply]
https://ar.vuejs.org/ done. Sorry for the long time waiting.

**Reply by Jinjiang** [reply]
Feel free to add a link to the English docs like this PR https://github.com/vuejs/docs/pull/2663

**Reply by boussadjra** [reply]
Thank you very much @Jinjiang

**Reply by Abdelaziz18003** [reply]
Thanks @Jinjiang we really appreciate that 👍
